controllers/userController.js

```diff
@@ -1,60 +1,88 @@
 
-const signUp = async (req, res) => {
-  try {
-    const { email, name, password, address, phone, birthdate } = req.body;
+const signUp = catchAsync(async (req, res) => {
+  const { email, name, password, address, phone, birthdate } = req.body;
 
-    if (!name || !password || !address || !phone || !email || !birthdate) {
-      const err = new Error("KEY_ERROR");
-      err.statusCode = 400;
-      throw err;
-    }
+  if (!name || !password || !address || !phone || !email || !birthdate) {
+    const err = new Error("KEY_ERROR");
+    err.statusCode = 400;
+    throw err;
+  }
+
+  await userService.signUp(email, name, password, address, phone, birthdate);
+  return res.status(200).json({ message: "SIGNUP_SUCCESS" });
+});
+
+const signIn = catchAsync(async (req, res) => {
+  const { email, password } = req.body;
 
-    await userService.signUp(email, name, password, address, phone, birthdate);
-    return res.status(200).json({ message: "SIGNUP_SUCCESS" });
-  } catch (err) {
-    return res.status(err.statusCode || 500).json({ message: err.message });
+  if (!email || !password) {
+    const err = new Error("KEY_ERROR");
+    err.statusCode = 400;
+    throw err;
   }
-};
 
-const signIn = async (req, res) => {
-  try {
-    const { email, password } = req.body;
+  const token = await userService.signIn(email, password);
 
-    if (!email || !password) {
-      const err = new Error("KEY_ERROR");
-      err.statusCode = 400;
-      throw err;
-    }
+  return res.status(200).json({ accessToken: token });
+});
 
-    const token = await userService.signIn(email, password);
+const emailCheck = catchAsync(async (req, res) => {
+  const { email } = req.body;
 
-    return res.status(200).json({ accessToken: token });
-  } catch (err) {
-    console.log(err);
-    return res.status(err.statusCode || 500).json({ message: err.message });
+  if (!email) {
+    const err = new Error("KEY_ERROR");
+    err.statusCode = 400;
+    throw err;
   }
-};
 
-const emailCheck = async (req, res) => {
-  try {
-    const { email } = req.body;
-
-    if (!email) {
-      const err = new Error("KEY_ERROR");
-      err.statusCode = 400;
-      throw err;
-    }
-
-    const mailCheck = await userService.mailCheck(email);
-    if (mailCheck === false) {
-      return res.status(200).json({ message: "사용가능한 이메일입니다." });
-    }
-    return res.status(409).json({ message: "사용 불가능한 이메일입니다." });
-  } catch (err) {
-    console.log(err);
-    return res.status(err.statusCode || 500).json({ message: err.message });
+  const mailCheck = await userService.mailCheck(email);
+  if (mailCheck === false) {
+    return res.status(200).json({ message: "사용가능한 이메일입니다." });
   }
-};
+  return res.status(409).json({ message: "사용 불가능한 이메일입니다." });
+});
 
-module.exports = { signUp, signIn, emailCheck };
+const getUserInfo = catchAsync(async (req, res) => {
+  const userId = req.user.id;
+  const result = await userService.userInfoGet(userId);
```

Comment:
```suggestion
  const result = await userService.getUserInfo(userId);
```
다 동일하게 작성해주세요. layer 별로 이름이 다를 필요는 없습니다. 오히려 가독성 및 유지 보수에 어려울 것 같아요.

Reply:
네 레이어별로 동일하게 전부 수정했습니다 !